docs/zos-native-os-linkage.md

```diff
@@ -0,0 +1,61 @@
+# z/OS Native OS Linkage Example
+
+These steps describe how to build and test this sample REST API in a way that
+uses JNI to call a native z/OS service - WTO.  You can follow these steps and modify them to
+call your own assembler / C API.
+
+These instructions assume that you follow the [z/OS deployment setup](./zos-deployment).
```

Comment:
Link is broken... looks like you're missing the extension (.md)

Reply:
Totally blaming VS Code for this one 😄 

Fixed.